packages/roger/package.json

```diff
@@ -3,13 +3,15 @@
   "version": "3.0.0",
   "description": "",
   "type": "module",
-  "main": "index.tsx",
+  "main": "dist/index.js",
   "scripts": {
-    "build": ":",
+    "build": "tsc",
     "build-decls": ":",
-    "start": "ts-node --esm --swc ./index.ts",
-    "typecheck": "tsc"
+    "typecheck": ":"
   },
+  "files": [
+    "dist/**/*.js"
+  ],
```

Comment:
Does this exclude `bin/run.js`?

Reply:
Not super sure tbh

Reply:
including `bin/run.js` and tested with `npm pack`:

```
npm notice
npm notice 📦  @penrose/roger@3.0.0
npm notice === Tarball Contents ===
npm notice 1.6kB  README.md
npm notice 47B    bin/run.js
npm notice 10.4kB dist/index.js
npm notice 9.8kB  dist/index.js.map
npm notice 44B    dist/types.js
npm notice 98B    dist/types.js.map
npm notice 6.4kB  dist/watch.js
npm notice 5.6kB  dist/watch.js.map
npm notice 1.0kB  package.json
npm notice === Tarball Details ===
npm notice name:          @penrose/roger
npm notice version:       3.0.0
npm notice filename:      penrose-roger-3.0.0.tgz
npm notice package size:  9.2 kB
npm notice unpacked size: 35.0 kB
npm notice shasum:        ba19590180cfd42aa5eb365f3db56c8f23121bc0
npm notice integrity:     sha512-M8+/vRrSM5RJv[...]FJIb65Cc3Nk3A==
npm notice total files:   9
npm notice
penrose-roger-3.0.0.tgz
```